### ISSUE TITLE
Pets information not stored correctly

### ISSUE THREAD

**Issue by brenoepics** [brenoepics/morningstar#922]
In GitLab by @minimario350 on Dec 12, 2020, 15:31

 

## Build Information

Emulator Version:  
Emulator Build (hash):

## Expected Behavior
Reloading a room, no matter the circunstances, shouldn't reset/rollback any pet's information.
## Current Behavior
Pets information gets reset every time the room gets reset.

## Steps to Reproduce
1. Buy a pet (Starts with level 1, 0/100 exp)
2. Place it in an empty room.
3. Reload the room.
4. Scratch it 3 times. Now it should have 30/100 exp.
5. Reload the room. The pet will roll back to 0/100 exp.

## Context (Environment)
Tried to train pets to get achievements and this causes it to be way more easy.

TL;DR
Reloading a room rolls back pet information.
## Possible Implementation
